poucave/html/js/modules.js

```diff
@@ -0,0 +1 @@
ion G(e,n){return z(function(){return e},n)}function J(e){var n=N.context[e.__c];if(!n)return e.__;var t=R(U++);return null==t.i&&(t.i=!0,n.sub(N)),n.props.value}function K(n,t){e.useDebugValue&&e.useDebugValue(t?t(n):n)}function Q(){A.some(function(n){if(n.__P)try{n.__H.u.forEach(X),n.__H.u.forEach(Y),n.__H.u=[]}catch(t){return e.__e(t,n.m),!0}}),A=[]}function X(e){e.p&&e.p()}function Y(e){var n=e.i();"function"==typeof n&&(e.p=n)}function Z(e,n){return!e||n.some(function(n,t){return n!==e[t]})}function ee(e,n){return"function"==typeof n?n(e):n}e.__r=function(e){F&&F(e),U=0,(N=e.__c).__H&&(N.__H.u.forEach(X),N.__H.u.forEach(Y),N.__H.u=[])},e.diffed=function(n){L&&L(n);var t=n.__c;if(t){var o=t.__H;o&&o.u.length&&(1!==A.push(t)&&M===e.requestAnimationFrame||((M=e.requestAnimationFrame)||function(e){var n,t=function(){clearTimeout(o),cancelAnimationFrame(n),setTimeout(e)},o=setTimeout(t,100);"undefined"!=typeof window&&(n=requestAnimationFrame(t))})(Q))}},e.__c=function(n,t){t.some(function(n){try{n.__h.forEach(X),n.__h=n.__h.filter(function(e){return!e.i||Y(e)})}catch(o){t.some(function(e){e.__h&&(e.__h=[])}),t=[],e.__e(o,n.m)}}),W&&W(n,t)},e.unmount=function(n){O&&O(n);var t=n.__c;if(t){var o=t.__H;if(o)try{o.t.forEach(function(e){return e.p&&e.p()})}catch(n){e.__e(n,t.m)}}};var ne=function(e,n,t,o){var r;n[0]=0;for(var _=1;_<n.length;_++){var u=n[_++],l=n[_]?(n[0]|=u?1:2,t[n[_++]]):n[++_];3===u?o[0]=l:4===u?o[1]=Object.assign(o[1]||{},l):5===u?(o[1]=o[1]||{})[n[++_]]=l:6===u?o[1][n[++_]]+=l+"":u?(r=e.apply(l,ne(e,l,t,["",null])),o.push(r),l[0]?n[0]|=2:(n[_-2]=0,n[_]=r)):o.push(l)}return o},te=new Map;function oe(n,t){var o,_,i,c,s,a;o=n,_=t,i=t.firstElementChild,e.__&&e.__(o,_),s=(c=i===r)?null:i&&i.__k||_.__k,o=f(p,null,[o]),a=[],w(_,(c?_:i||_).__k=o,s||u,u,void 0!==_.ownerSVGElement,i&&!c?[i]:s?null:l.slice.call(_.childNodes),a,i||u,c),S(a,o)}var re=function(e){var n=te.get(this);return n||(n=new Map,te.set(this,n)),(n=ne(this,n.get(e)||(n.set(e,n=function(e){for(var n,t,o=1,r="",_="",u=[0],l=function(e){1===o&&(e||(r=r.replace(/^\s*\n\s*|\s*\n\s*$/g,"")))?u.push(0,e,r):3===o&&(e||r)?(u.push(3,e,r),o=2):2===o&&"..."===r&&e?u.push(4,e,0):2===o&&r&&!e?u.push(5,0,!0,r):o>=5&&((r||!e&&5===o)&&(u.push(o,0,r,t),o=6),e&&(u.push(o,e,0,t),o=6)),r=""},i=0;i<e.length;i++){i&&(1===o&&l(),l(i));for(var c=0;c<e[i].length;c++)n=e[i][c],1===o?"<"===n?(l(),u=[u],o=3):r+=n:4===o?"--"===r&&">"===n?(o=1,r=""):r=n+r[0]:_?n===_?_="":r+=n:'"'===n||"'"===n?_=n:">"===n?(l(),o=1):o&&("="===n?(o=5,t=r,r=""):"/"===n&&(o<5||">"===e[i][c+1])?(l(),3===o&&(u=u[0]),o=u,(u=u[0]).push(2,0,o),o=0):" "===n||"\t"===n||"\n"===n||"\r"===n?(l(),o=2):r+=n),3===o&&"!--"===r&&(o=4,u=u[0])}return l(),u}(e)),n),arguments,[])).length>1?n:n[0]}.bind(f);export{f as h,re as html,oe as render,h as Component,T as createContext,V as useState,I as useReducer,q as useEffect,B as useLayoutEffect,$ as useRef,j as useImperativeHandle,z as useMemo,G as useCallback,J as useContext,K as useDebugValue};
```

Comment:
oh sorry i should rename this to `htm_preact.module.js` which I copied verbatim from here:
https://unpkg.com/htm/preact/standalone.module.js. because we want to import from it as a module we cannot use the concatenation that we are using for the other libraries.